src/FileSystem/FilePathHelper.php

```diff
@@ -45,6 +45,15 @@ public function relativePath(string $fileRealPath): string
         return $this->relativeFilePathFromDirectory($fileRealPath, getcwd());
     }
 
+    public function absolutePath(string $fileRealPath): string
+    {
+        if ($this->filesystem->isAbsolutePath($fileRealPath)) {
+            return $fileRealPath;
+        }
+
+        return realpath($fileRealPath);
+    }
```

Comment:
this is not needed, just use `realpath()` on per my suggestion on https://github.com/rectorphp/rector-src/pull/6293/files#r1749022916 is enough